### DISCUSSION TITLE
카카오-선착순 마감, 정상 작동 여부

### DISCUSSION THREAD

**Discussion by Jnthn-K** [kimytsc/covid-rest-vaccine-macro#39] [Q&A] [answered]
네이버와 카카오 둘다 잘 작동하는 것 같은데요.
네이버는 리프레쉬 레이트가 5초 정도로 잡히는 것 같고,
카카오는 500미리세컨드 까지 내려도. 문제없이 잘 돌아가는 것 같습니다.

아침부터 돌리고 있는 중인데요,
네이버는 딜레이가 길다보니까, 선착순 마감되었다는 메시지를 보게 되는것이 의심스럽지가 않은데...
카카오도 동일하게 선착순 마감되었다는 메시지가 자꾸 나오네요.

콘솔쪽에는 아래와 같이 에러메시지가 표출됩니다. 세부적인건 생략했습니다.
![image](https://user-images.githubusercontent.com/88519543/128815092-d32cb1f3-d21c-4caf-bfe1-2791d56c9900.png)

정상적인 작동인것인지 궁금합니다!

**Comment by kimytsc** [comment, marked as answer]
/api/v2/reservation 403 로그는 예약 시도 실패시 나타나는 로그입니다.

카카오는 예약 시도 실패시 403 상태코드로 응답을 주는데, 이 과정에서 무조건적으로 `Failed to load resource: the server responded with a status of 403 ()`이 로그에 노출되게 됩니다.

화면에서 병원명과 실패사유가 변경된다면, 정상적이며, 적어주신 내용으로 보아 정상으로 판단됩니다.

**Reply by Jnthn-K** [reply]
답변 감사합니다!
